apps/examples/testcase/tc_common.h

```diff
@@ -133,7 +133,25 @@ extern int total_fail;
 	} \
 }
 
-#define TC_ASSERT_LEQ(api_name, var, ref) TC_ASSERT_LEQ_CLEANUP(api_name, var, ref, )
+#define TC_ASSERT_LEQ(api_name, var, ref) TC_ASSERT_LEQ_CLEANUP(api_name, var, ref,)
+
+#define TC_ASSERT_NAN_CLEANUP(api_name, var, ref, freeResource) \
```

Comment:
Is it not possible to compare using boolean check values instead of this ?

Reply:
If we compare using boolean  check then also we need to print the result, so instead of duplicating the same effort in all the functions, I have implemented the common API for that.
And I think  we follow the same method for other checks also like LEQ and EQ for the other values.

Reply:
Updated the logic to eliminate the TC_ASSERT_NAN macro